guides/source/getting_started.md

```diff
@@ -83,8 +83,6 @@ proper prerequisites installed. These include:
 
 * Ruby
 * SQLite3
-* Node.js
-* Yarn
```

Comment:
FYI this guide is being re-written here https://github.com/rails/rails/pull/43957

Reply:
Oh, cool. Thanks for letting me know @ghiculescu. I'll close my PR! 🙂